src/components/home/ShowcaseGrid/ShowcaseGrid.tsx

```diff
@@ -106,13 +106,25 @@ const Column: React.FC<ColumnProps> = ({className, posters}) => (
         )
       }
 
-      return (
-        <div className="border border-gray-400 rounded-sm" key={i}>
-          <div style={{width: 320, height: 424}}></div>
-        </div>
-      )
+      return <PosterPlaceholder key={i} />
     })}
   </div>
 )
 
+const PosterPlaceholder: React.FC = () => (
+  <div className="border border-gray-400 rounded-sm">
+    <div
+      style={{width: 320, height: 424}}
+      className="flex flex-col justify-center"
+    >
+      <h2
+        style={{width: 320}}
+        className="text-center text-2xl opacity-40 cursor-default"
+      >
+        Your Poster Here...
```

Comment:
Keep in mind this will show up in screen readers. Perhaps, we better add an aria-hidden for these placeholder posters.

Reply:
Sounds good to me! Thanks for the great suggestion :+1: